File: test_db.js
const fs = require('fs');
const html = fs.readFileSync('ML63.html', 'utf8');
const urlMatch = html.match(/supabase\.createClient\('([^']+)',\s*'([^']+)'\)/);
if (!urlMatch) { console.error("No url/key"); process.exit(1); }
const url = urlMatch[1];
const key = urlMatch[2];

fetch(`${url}/rest/v1/movimientos?select=fecha,concepto,importe,saldo&order=fecha.desc&limit=10`, {
  headers: { 'apikey': key, 'Authorization': `Bearer ${key}` }
}).then(r => r.json()).then(console.log).catch(console.error);

File: test_db.py
import urllib.request, json, ssl

url = 'https://byqtsuskdbgwpyvyiprc.supabase.co'
key = 'sb_publishable_LVuLdmu3YUa0HhMZGdBKtg_VHPZUizx'

req = urllib.request.Request(f"{url}/rest/v1/movimientos?select=fecha,concepto,importe,saldo&order=fecha.desc&limit=10", headers={
    'apikey': key,
    'Authorization': f'Bearer {key}'
})
ctx = ssl._create_unverified_context()
with urllib.request.urlopen(req, context=ctx) as res:
    print(json.dumps(json.loads(res.read().decode()), indent=2))
